content/en/docs/instrumentation/js/_index.md

```diff
@@ -15,6 +15,6 @@ weight: 20
 ## Further Reading
 
 - [OpenTelemetry for JavaScript on GitHub](https://github.com/open-telemetry/opentelemetry-js)
-- [Getting Started]({{< relref "getting-started" >}})
-- [API Reference](https://open-telemetry.github.io/opentelemetry-js-api)
+- [Getting Started](getting-started/)
+- [API Reference](https://open-telemetry.github.io/opentelemetry-js/modules/_opentelemetry_api.html)
```

Comment:
@open-telemetry/javascript-approvers - is this a valid link for **API Reference** or should I just combine it with ** SDK Reference** as in:

> - [API and SDK reference](https://open-telemetry.github.io/opentelemetry-js)

Linking to https://open-telemetry.github.io/opentelemetry-js?

Reply:
This should be https://open-telemetry.github.io/opentelemetry-js/

(it's the same link we use in the API node for the sidebar0

Reply:
If we'll be using the same link then the two entries should be combined, as I suggested above. WDYT?

Reply:
Yeah, let's just combine it.

There's a _whole thing_ I have in my head about how there's a full API reference for the thing you download and install (the SDK, which includes the API), and the API reference for the "api", which is literally just the opentelemetry api itself, the thing designed to work with any arbitrary instantiated SDK. It's the right technical design but being this precise for end-users who primarily install an SDK anyways just makes things confusing.